.heroku/vendor/pygobstones-lang

```diff
@@ -1 +1 @@
-Subproject commit 141d657e1317132a215f54cec952580032d86f6d
+Subproject commit 34e62b00bf7e0ce23094be7c9a8cc6237c6ebdde
```

Comment:
Great! We should also update [stones-spec](https://github.com/uqbar-project/stones-spec) version to match this one.

However, as I told you some days ago, depending of PyGobstones in both projects doesn't look fine for me. It's ok for now, but we should address it soon. What do you think @flbulgarelli?